services/drupal/web/themes/epa_theme/source/_patterns/05-components/back-to-top/back-to-top.twig

```diff
@@ -1,4 +1,4 @@
-<a href="#{{ top_element }}" class="back-to-top" title="{{ text }}" aria-hidden="true">
+<a href="#{{ top_element }}" class="back-to-top" title="{{ text }}" aria-hidden="true" tabindex="-1">
```

Comment:
This will permanently remove the back to top link from being tabbed to, which we don’t want. We should add the disabled attribute here instead, then remove it in JS when appropriate.